client/src/elm/Backend/Update.elm

```diff
@@ -3919,11 +3924,23 @@ generatePrenatalAssessmentMsgs currentDate language isChw activePage updateAsses
                                ]
 
             else
-            -- Logic for nurse
+            -- Logic for nurse.
             if
                 mandatoryActivitiesCompleted
             then
                 let
+                    _ =
+                        Debug.log "diagnosesBefore" diagnosesBefore
+
+                    _ =
+                        Debug.log "diagnosesAfter" diagnosesAfter
+
+                    _ =
+                        Debug.log "addedDiagnoses" addedDiagnoses
+
+                    _ =
+                        Debug.log "initialEncounterMsgs" initialEncounterMsgs
+
```

Comment:
@anvmn Probably can be deleted, right?

Reply:
@mariano-dagostino well, as this is ongoing development, and more and more diagnoses are added, these logs are useful untill feature is completed.
However, to keep the code 'formal', I'll delete them.